README.md

```diff
@@ -23,140 +25,154 @@
+is intended to be used from OPS on the jumpbox (Upp Jumpbox p).
+The script sits in `/usr/local/bin` and is invoked by `/etc/skel/.bashrc`
+everytime a user is connected. The purpose of the script is similar - it will
+download and merge the kubeconfigs for OPS and will get `kubectx` tool
+on the jumpbox.
 
-```shell
-PROD_ACCOUNT_CLUSTERS=(
-  eks-pac-staging-eu
-  eks-pac-staging-us
-
-)
-TEST_ACCOUNT_CLUSTERS=(
-  eks-publish-staging-eu
-  eks-delivery-staging-eu
-)
-```
+### Connect to EKS cluster (Step by step guide)
 
-1. Execute `update-eks-kubeconfig.sh`
+1. Checkout the kubectl-login repo and get into the proper folder:
 
-```shell
-bash update-eks-kubeconfig.sh
-```
+    ```shell
+    git clone git@github.com:Financial-Times/kubectl-login.git
+    cd update-eks-kubeconfig/
+    ```
 
-Restricted VPN is now no longer needed.
+1. Check the script `update-eks-kubeconfig.sh` and update the EKS cluster names
+  for PROD and TEST accounts, if needed:
 
-1. Export KUBECONFIG
+    ```shell
+    PROD_ACCOUNT_CLUSTERS=(
+      ...
+    )
+    TEST_ACCOUNT_CLUSTERS=(
+      ...
+    )
+    ```
 
-```shell
-export-eks-kubeconfig
-```
+    How to check what are the EKS clusters' names?
 
-or if you miss the alias
+    - Log into the [AWS console](https://awslogin.in.ft.com/adfs/ls/IdpInitiatedSignOn.aspx?loginToRp=urn:amazon:webservices).
+    - Check accordingly test and prod accounts.
+    - Look for `EKS` and check the cluster names there.
 
-```shell
-export KUBECONFIG=$HOME/.kube/eks-kubeconfig
-```
+1. Once happy with the cluster names setup:
 
-1. Install [kubectx](https://github.com/ahmetb/kubectx)
+   - Make sure that you are connected to the Restricted VPN.
+   - Then execute `update-eks-kubeconfig.sh`
+     (you should be in its folder):
 
-```shell
-brew install kubectx
-```
+    ```shell
+    bash update-eks-kubeconfig.sh
+    ```
 
-1. Run kubectx
+    - Restricted VPN is now no longer needed after this step.
 
-```shell
-laptop$ kubectx
-eks-delivery-test-eu
-eks-pac-test-eu
-laptop$
-```
+1. Set KUBECONFIG to its proper EKS value
 
-1. Connect to EKS cluster
+    ```shell
+    export KUBECONFIG=$HOME/.kube/eks-kubeconfig
+    ```
 
-```shell
-laptop$ kubectx eks-delivery-test-eu
-```
+1. Install [kubectx](https://github.com/ahmetb/kubectx) and run it:
+
+    ```shell
+    brew install kubectx
+    ```
 
-1. Profit
+    ```shell
+    kubectx
+    ```
+
+1. Connect to an EKS cluster by specifying its name:
+
+    ```shell
+    kubectx eks-publish-dev-eu
+    ```
+
+1. Check that you are able to execute commands:
+
+    ```shell
+    kubectl get pods
+    ```
+
+### Navigation between K8S and EKS
+
+#### Aliases
+
+The main difference is the `KUBECONFIG` value.
+So for easier access to both K8S and EKS of clusters,
+you can set up some aliases to quickly change that:
 
 ```shell
-kubectl get pods
+alias k8s='export KUBECONFIG=$HOME/content-k8s-auth-setup/kubeconfig'
```

Comment:
The aliases are too generic and may already be used by a user.
I think that the previous version was better: `export-eks-kubeconfig` / `export-k8s-kubeconfig`

Reply:
Those are way too long for aliases for my taste but I see your point. I've updated the example and added a note. :)